primefaces/pom.xml

```diff
@@ -607,6 +606,7 @@
                         <!-- uncompressed widgets -->
                         <resourcesSet>
                             <includes>
+                                <include>chart/**</include>
```

Comment:
does it get minified then?

Reply:
No I moved this to the unminified section so it gets combined but not minified.

Reply:
why not minify it?

Reply:
I assume you're refering to this: https://github.com/primefaces/primefaces/pull/10871#issuecomment-1774091993 and my PR contained the workaround. Ideally everything should be minified (whether it's small or big),

Reply:
Yes so I saw what you did nd it does not work.  You tried to compress `2-chartjs-widget.js` and then later combine them into `charts.js` with the 2 minified file sand `2-widget.js` but that won't work.  It Just combines them but the widget is still uncompressed.

Reply:
[chart.js.txt](https://github.com/primefaces/primefaces/files/13975438/chart.js.txt)

I can see 3 files in here and everything look compressed, am I missing something?

Reply:
Let me try again but that is not the behavior I am seeing inside the final JAR.  Maybe I didn't do a `clean`